sorc/ncep_post.fd/WRFPOST.f

```diff
@@ -648,16 +616,10 @@ PROGRAM WRFPOST
           IF(MODELNAME == 'NCAR' .OR. MODELNAME == 'RAPR') THEN
             print*,'CALLING INITPOST TO PROCESS NCAR NETCDF OUTPUT'
             CALL INITPOST
-          ELSE IF(MODELNAME == 'NMM') THEN
```

Comment:
@WenMeng-NOAA @fossell Is DTC still supporting WRF NMM users using NetCDF output?

Reply:
No, DTC stopped support of WRF-NMM a few years ago. It's ok to remove this.

Reply:
Thanks for clarifying. I will remove it.